src/index.html

```diff
@@ -17,6 +17,10 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch">
+      <div class="stopwatch__minutes stopwatch__minutes--speed-up"></div>
+      <div class="stopwatch__sekundes stopwatch__sekundes--speed-up"></div>
```

Comment:
There is a typo in the class name 'stopwatch__sekundes'. It should be 'stopwatch__seconds'. Please correct it.